code/_onclick/hud/rendering/render_plate.dm

```diff
@@ -393,10 +393,7 @@
  * Other vars such as alpha will automatically be applied with the render source
  */
 /atom/movable/screen/plane_master/proc/generate_render_relays()
-#if MIN_COMPILER_VERSION > 516
-	#warn Fully change default relay_loc to "1,1"
-#endif
-	var/relay_loc = home?.relay_loc || "CENTER"
+	var/relay_loc = home?.relay_loc || "SCREEN_SOUTHWEST"
```

Comment:
why different from the warn?

Reply:
https://github.com/tgstation/tgstation/pull/84035#issuecomment-2725541932